web/elm/src/Dashboard/Dashboard.elm

```diff
@@ -267,7 +271,16 @@ handleDeliveryBody delivery ( model, effects ) =
             )
 
         ClockTicked FiveSeconds _ ->
-            ( model, effects ++ [ FetchData, FetchPipelines ] )
+            ( { model | isDataLoading = True }
+            , effects
+                ++ FetchPipelines
```

Comment:
Should we also ensure we don't pile on the list pipelines calls? Or is there little value in that?

Reply:
a pipeline is much smaller than a job, and there are fewer pipelines than jobs. those pipelines are getting fetched for the benefit of the sidebar, I think